magma/bits.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, *args, **kwargs):
             self.ts = args[0].ts[:]
             # zext for promoting width
             for i in range(len(self) - len(args[0])):
-                self.ts.append(GND)
+                self.ts.append(Bit.GND)
```

Comment:
Should this be Bit(0) for consistency with elsewhere?

Reply:
Fixed